angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/weather.html

```diff
@@ -26,13 +37,16 @@
                         <div class="col-xs-3 day">
                           <p>Now</p>
                           <img ng-src="{{config.iconPrefix}}{{weather.currentWeather.imgName}}{{config.iconPostfix}}" title="{{weather.currentWeather.condition}}" alt="{{weather.currentWeather.condition}}">
-                          <p>{{weather.currentWeather.temperature}}&deg; {{weather.temperatureUnit}}</p>
+                          <p ng-show="!showMetric" >{{weather.currentWeather.temperature | number:0}}&deg; {{weather.temperatureUnit}}</p>
```

Comment:
Idea: what if the temperature unit ( "C" or "F") *were* the toggle to switch between units?  Don't like degrees F?  Click the F and switch it to C.

Would be way, way more subtle.  Maybe too subtle.  Some users wouldn't notice it.  But then most users accept the default units, and anyone who cares enough to raise it to the helpdesk could be walked through clicking the unit to change it.